### DISCUSSION TITLE
Log4j vulnerability risk ?

### DISCUSSION THREAD

**Discussion by steevi2307** [red6/pdfcompare#116] [Q&A] [answered]
Hi,

Do you think there is a risk using the pdfcompare file as we heard about the log4j vulnerability today.
I see some references about these :

import org.apache.pdfbox.pdmodel.PDDocument;
import org.apache.pdfbox.pdmodel.PDPage;
import org.apache.pdfbox.pdmodel.ResourceCache;
import org.apache.pdfbox.pdmodel.common.PDRectangle;
import org.apache.pdfbox.rendering.PDFRenderer;
**import org.slf4j.Logger;
import org.slf4j.LoggerFactory;**

Thanks,
Steven

**Comment by finsterwalder** [comment, marked as answer]
PdfCompare uses slf4j and logback not Log4J, so there is no risk.

**Reply by steevi2307** [reply]
Thanks :)
